src/app/view/components/dot-listing-data-table/dot-listing-data-table.component.ts

```diff
@@ -201,6 +202,14 @@ export class DotListingDataTableComponent implements OnInit {
         }
     }
 
+    /**
+     * Check if display results are filtered.
+     * @memberof ListingDataTableComponent
+     */
+    handleFilter(): void {
+        this.isContentFiltered = this.dataTable.filters.hasOwnProperty('global') ? true : false;
```

Comment:
`hasOwnProperty` returns a boolean, so no need of a ternary

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty